docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -1,43 +1,73 @@
+
+Application services:
+
+- [App Service Configurable](https://snapcraft.io/edgex-app-service-configurable)
 
-[Snaps](https://snapcraft.io/docs) are a hybrid of traditional Linux packages and containers. Snaps can be installed on any
-Linux distro with snap support enabled, although full confinement currently requires some flavor of Ubuntu (Desktop/Server
-or [Core](https://ubuntu.com/core/docs)).
+Device services:
 
-Canonical publishes snaps (amd64 & arm64) for each release of EdgeX to the Snap Store. In contrast to docker deployment, all core,
-security, support, and runtime dependencies are provided in a single snap called [edgexfoundry](https://snapcraft.io/edgexfoundry).
-Additional snaps are available for [App Service Configurable](https://snapcraft.io/edgex-app-service-configurable), as well as the
-standard set of EdgeX reference device services (see [list](#device-service-snaps) below). The edgexfoundry snap also includes Device Virtual to allow users
-to experiment with EdgeX without installing additional snaps.
+- [Device Camera](https://snapcraft.io/edgex-device-camera)
+- [Device Modbus](https://snapcraft.io/edgex-device-modbus)
+- [Device MQTT](https://snapcraft.io/edgex-device-mqtt)
+- [Device REST](https://snapcraft.io/edgex-device-rest)
+- [Device SNMP](https://snapcraft.io/edgex-device-snmp)
+- [Device Grove](https://snapcraft.io/edgex-device-grove)
 
-## Installing EdgeX Foundry Jakarta Snaps
+Other EdgeX snaps do exist on the public Snap Store ([search by keyword](https://snapcraft.io/search?q=edgex)) or private stores under brand accounts.
 
-The Snap Store allows multiple versions of a snap to be published to version-specific tracks. If not specified, snaps are installed
-from the `latest/stable` track. 
+## Installing the `edgexfoundry` snap
 
-You can see the current snap tracks and revisions available for your machine's architecture by running the command:
+The Snap Store allows access to multiple versions of the snap using [channels](https://snapcraft.io/docs/channels). If not specified, snaps are installed
+from the default `latest/stable` channel. 
+
+You can see the current snap channels available for your machine's architecture by running the command:
 
 ```bash
 snap info edgexfoundry
 ```
 
-In order to install Jakarta versions of the EdgeX snaps, you need to specify the `--channel=2.1` command-line option:
+In order to install a specific version of the snap by setting the `--channel` flag.
+For example, to install the Jakarta (2.1) release:
 
 ```bash
 sudo snap install edgexfoundry --channel=2.1
 ```
 
-!!! Note
-    The snap has only been tested on Ubuntu Desktop/Server LTS 18.04/20.04, as well as Ubuntu Core versions 18 and 20.
+To install the latest beta:
+```bash
+sudo snap install edgexfoundry --channel=latest/beta
+# or using the shorthand
+sudo snap install edgexfoundry --beta
+```
 
-## Using the EdgeX Snap
+Replace `beta` with `edge` to get the latest nightly build!
```

Comment:
I think adding a new line here, will make the installation section clearly separated from the following post-installation section.

Or make this sentence as a note?
!!! Note
Replace `beta` with `edge` to get the latest nightly build!

Reply:
I added a horizontal line after it.